src/components/Menu.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div id="drawerSection">
+    <label for="nav-input">
+      <div id="drawerButton">
+        <img
+          id="close"
+          src="../assets/close.png"
+        >
+      </div>
+    </label>
+
+    <ul class="drawerMenuSection">
+      <li class="drawerMenuItem">
+        <a
+          href="#main"
+          class="drawerMenuItemLink"
+          @click="clickSmoothScroll('#mainSection')"
+        >Home</a>
+      </li>
+      <li class="drawerMenuItem">
+        <a
+          href="#about"
+          class="drawerMenuItemLink"
+          @click="clickSmoothScroll('#aboutSection')"
+        >About me</a>
+      </li>
+      <li class="drawerMenuItem">
+        <a
+          href="#skill"
+          class="drawerMenuItemLink"
+          @click="clickSmoothScroll('#skillSection')"
+        >Skill Sets</a>
+      </li>
+      <li class="drawerMenuItem">
+        <a
+          href="#vision"
+          class="drawerMenuItemLink"
+          @click="clickSmoothScroll('#visionSection')"
+        >Vision</a>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+export default {
+  methods: {
+      clickSmoothScroll (Section) {
```

Comment:
method化すごい良いです！！！！
```suggestion
      clickSmoothScroll (section) {
```
変数名もキャメルケースになっているとなお良いです！